src/smart-components/workflow/add-groups/add-workflow-wizard.js

```diff
@@ -66,28 +36,52 @@ const AddWorkflow = () => {
   };
 
   return (
-    <Wizard
-      title={ 'Create approval process' }
-      isOpen
-      onClose={ onCancel }
-      onSave={ onSave  }
-      onNext={  onNext }
-      steps={ steps }
+    <FormRenderer
+      showFormControls={ false }
+      onSubmit={ onSave }
+      onCancel={ onCancel }
+      schema={ {
```

Comment:
Can we store the schema (to ref maybe?) to avoid unnecessarily redeclaring this with renders?

Reply:
I extracted the schema into separate function.

I tried to make a constant from it but patternfly proptypes do not allow that.

However, I don't think it's not necessary to avoid re-declaring here - there are no props and no state. The wizard should not be re-rendered at all.